spring-integration-amqp/src/test/java/org/springframework/integration/amqp/outbound/AsyncAmqpGatewayTests.java

```diff
@@ -178,6 +213,7 @@ public void handleMessage(Message<?> message) throws MessagingException {
 		assertEquals("nacknack", ack.getHeaders().get(AmqpHeaders.PUBLISH_CONFIRM_NACK_CAUSE));
 		assertEquals(false, ack.getHeaders().get(AmqpHeaders.PUBLISH_CONFIRM));
 
+		asyncTemplate.stop();
 		receiver.stop();
 	}
```

Comment:
I wonder if we have to `ccf.destroy()` in the of test as well.
At least I see that in the SA tests, when we don't use application context.